util/url.go

```diff
@@ -27,3 +27,7 @@ func CheckURL(str string) error {
 	}
 	return nil
 }
+
+func HttpsUrl(url string) string {
```

Comment:
super nit
this might be uncessary but public functions technically need a comment 😂